app/src/main/java/uw/hcrlab/kubi/robot/Robot.java

```diff
@@ -229,6 +451,10 @@ public void say(String msg, String language, int speed) {
         }
     }
 
+    public void shutup() {
+        tts.stop();
+    }
+
```

Comment:
I thought this might actually not be very nice of me... I almost changed it haha